lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -1753,6 +1753,21 @@ defmodule Kernel.SpecialForms do
   catch values also support `:error`, as in Erlang, although it is
   commonly avoided in favor of raise/rescue control mechanisms.
 
+  ## After clauses
+
+  An `after` clause allows you to define cleanup logic that will be invoked both
+  when the tried block of code succeeds and also when an error is raised. Note,
+  however, that if a linked process exits in the `try` block, the process will
+  immediately exit and your `after` clause will not get executed.
+
+      {:ok, file} = File.open "sample", [:utf8, :write]
```

Comment:
This can be a tricky example because, the file process is linked to the parent process, so in this case the file will be cleaned up anyway. The same applies to ports, ets, etc.